src/XRP/protobuf-wrappers/xrp-issued-currency.ts

```diff
@@ -1,3 +1,9 @@
+// TODO: friends - should we consider renaming this class as `XrplIssuedCurrency`?  (I mean... all of these wrapper classes should really
```

Comment:
yeah, i think that's a good thing to do - definitely a confusing name. 

want to make a new task to track it? doesn't seem like the most pressing thing ever but a good chore for sure. i'm happy to do it

Reply:
++

Reply:
i'll make a task!